extra_exercises/reverse_linked_list.py

```diff
@@ -0,0 +1,33 @@
+"""Print a linked list in reverse order. Follow-up: try to do it without additional memory (simple variables are fine)."""
+from linked_list.linked_list import LinkedList
+
+
+def reverse_with_list(linked_list):
+    reverse_linked = []
+    current = linked_list.head
+    while current.next is not None:
+        reverse_linked.insert(0, current.item)
+        current = current.next
+    reverse_linked.insert(0, current.item)
```

Comment:
you could avoid this additional insert by checking the while loop for `current is not None` instead